util/include/memory.hpp

```diff
@@ -1,6 +1,6 @@
 // Concord
 //
-// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+// Copyright (c) 2018-2022 VMware, Inc. All Rights Reserved.
```

Comment:
The name of this file is misleading, I would define its definitions where you use them (as far as I can see it is only in util/include/openssl_crypto.hpp

Reply:
It is an extension of std::memory, which lets you define unique pointers with custom deleters with relative ease.
It is used only once because it was implemented in this branch, the other crypto related branch will use it as well, and many places in our code can be refactored to use it and save up code lines.
openssl_crypto.hpp is a user of this generic code, Pankaj's branch will likely use it too.